src/main/java/Duke.java

```diff
@@ -1,10 +1,531 @@
+            + "\u2588\u2557\u255a\u2550\u2550\u2588\u2588\u2554\u2550\u2550\u255d\u2588\u2588" + "\u2551\u2588\u2588"
+            + "\u2554\u2550\u2550\u2550\u2550\u255d" + NEWLINE + "   \u2588" + "\u2588\u2551   \u2588\u2588\u2551"
+            + "   \u2588\u2588\u2551\u2588\u2588\u2551   " + "\u2588\u2588\u2551   \u2588\u2588\u2551   "
+            + "\u2588\u2588\u2551\u2588\u2588\u2588\u2588" + "\u2588\u2557  " + NEWLINE + "   \u2588\u2588\u2551 "
+            + "  \u2588\u2588\u2551   " + "\u2588\u2588\u2551\u2588\u2588\u2551   \u2588\u2588\u2551   "
+            + "\u2588\u2588\u2551   " + "\u2588\u2588\u2551\u2588\u2588\u2554\u2550\u2550\u255d  " + NEWLINE + " "
+            + "  \u2588" + "\u2588\u2551   \u255a\u2588\u2588\u2588\u2588\u2588\u2588\u2554\u255d\u255a\u2588\u2588"
+            + "\u2588\u2588\u2588\u2588\u2554\u255d   \u2588\u2588\u2551   " + "\u2588\u2588\u2551\u2588"
+            + "\u2588\u2588\u2588\u2588\u2588\u2588\u2557" + NEWLINE + "   \u255a" + "\u2550\u255d  "
+            + "  \u255a\u2550\u2550\u2550\u2550\u2550\u255d  " + "\u255a\u2550\u2550\u2550\u2550\u2550"
+            + "\u255d    \u255a\u2550\u255d   " + "\u255a\u2550\u255d\u255a\u2550\u2550\u2550\u2550\u2550"
+            + "\u2550\u255d" + NEWLINE;
+    public static final String SIMPLE_TOOTIE_LOGO =
+            " _____           _   _      " + NEWLINE + "|_   _|         | | (_)     " + NEWLINE + "  | | ___  " +
+            " ___ | |_ _  ___ " + NEWLINE + "  | |/ _ \\ / _ \\| __| |/ _ \\" + NEWLINE + "  | | (_) | (_) | |_| " +
+            "|  __/" + NEWLINE + "  \\_/\\___/ \\___/ \\__|_|\\___|" + NEWLINE;
+
+    // is complete indicator symbols
+    public static final String TICK_SYMBOL = "[\u2713]";
+    public static final String CROSS_SYMBOL = "[\u2717]";
+
+    // Emoticons in unicode
+    public static final String SPARKLY_EMOTICON =
+            "\u0028\uff89\u25d5\u30ee\u25d5\u0029\uff89\u002a\u003a\uff65\uff9f\u2727";
+    public static final String CONFUSED_EMOTICON = "\u0028\u30fb\u2227\u2010\u0029\u309e";
+    public static final String HAPPY_EMOTICON = "\uff08\u00b4\u30fb\u03c9\u30fb \u0060\uff09";
+    public static final String FLOWER_SMILE_EMOTICON = "(\u25e0\u203f\u25e0\u273f)";
+
+    public static final String SPARKLY_TEXT_DIVIDER = "\u2500\u2500\u2500\u2500\u2500\u2500\u2500 "
+            + "\u2731\u002a\u002e\uff61\u003a\uff61" + "\u2731\u002a\u002e\u003a\uff61\u2727\u002a\u002e\uff61"
+            + "\u2730\u002a\u002e\u003a\uff61\u2727" + "\u002a\u002e\uff61\u003a\uff61\u002a\u002e\uff61\u2731"
+            + " " + "\u2500\u2500\u2500\u2500\u2500" + "\u2500\u2500";
+
+    public static final int MAX_TASKS = 100;
+    public static final String TASKNAME_MARKER = "t/";
+    public static final String STARTTIME_MARKER = "s/";
+    public static final String ENDTIME_MARKER = "e/";
+    public static final String DUEDATE_MARKER = "d/";
```

Comment:
I think you can consider splitting the string, e.g. TASKNAME -> TASK_NAME to improve readability.